models/BaseConfig.cfc

```diff
@@ -344,6 +344,10 @@ component accessors="true" {
 	property name='requestQueueTimeout' type='numeric' _isCFConfig=true;
 	// Request Queue Timeout Page
 	property name='requestQueueTimeoutPage' type='string' _isCFConfig=true;
+	// Enable request queue on Lucee
+	property name='requestQueueEnable' type='boolean' _isCFConfig=true;
+	// Maximum number of concurrent requests on Lucee
+	property name='maxRequests' type='numeric' _isCFConfig=true;
```

Comment:
@bferdinandus Wouldn't we want to use the existing setting `maxTemplateRequests` for this?  

We already have the following for Adobe:
* `maxTemplateRequests` - Seems to be the equivalent setting
* `maxFlashRemotingRequests` - I don't think Lucee has this
* `maxWebServiceRequests` - Lucee doesn't have websockets
* `maxCFCFunctionRequests` - I have a feeling, Lucee counts these as regular HTTP requests and bundles them in bullet #1
* `maxReportRequests` - Lucee doesn't have report requests
* `maxCFThreads` - I don't think Lucee limits CFThreads at all